articles/service-fabric/service-fabric-get-started-mac.md

```diff
@@ -28,77 +28,39 @@ ms.author: saysa
 You can build Service Fabric applications to run on Linux clusters using Mac OS X. This article covers how to set up your Mac for development.
 
 ## Prerequisites
-Service Fabric does not run natively on OS X. To run a local Service Fabric cluster, we provide a pre-configured Ubuntu virtual machine using Vagrant and VirtualBox. Before you get started, you need:
+Service Fabric does not run natively on OS X. To run a local Service Fabric cluster, we provide a pre-configured Docker image. Before you get started, you need:
 
-* [Vagrant (v1.8.4 or later)](http://www.vagrantup.com/downloads.html)
-* [VirtualBox](http://www.virtualbox.org/wiki/Downloads)
+* [Docker](https://docs.docker.com/docker-for-mac/install/)
 
->[!NOTE]
-> You need to use mutually supported versions of Vagrant and VirtualBox. Vagrant might behave erratically on an unsupported VirtualBox version.
->
-
-## Create the local VM
-To create the local VM containing a 5-node Service Fabric cluster, perform the following steps:
+## Download and start the docker image.
+To start the docker image having a pre-configured 5-node Service Fabric cluster, perform the following steps:
 
-1. Clone the `Vagrantfile` repo
+1. Download and run the pre-configured docker image
 
     ```bash
-    git clone https://github.com/azure/service-fabric-linux-vagrant-onebox.git
+    sudo docker run -it -p 19080:19080 servicefabricoss/service-fabric-onebox bash
     ```
-    This steps downloads the file `Vagrantfile` containing the VM configuration along with the location the VM is downloaded from.  The file points to a stock Ubuntu image.
+    This steps downloads the the docker image and runs it.
 
-2. Navigate to the local clone of the repo
+2. Setup the Service Fabric
 
     ```bash
-    cd service-fabric-linux-vagrant-onebox
+    ./setup.sh
```

Comment:
A line after this which says. This script is provided for convenience for installing the Service Fabric runtime and the Service Fabric common SDK along with sfctl CLI.  Running the script assumes you agree to the licenses for all the software dependencies that are being installed.